bindings/rust/s2n-tls-sys/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "s2n-tls-sys"
 description = "A C99 implementation of the TLS/SSL protocols"
-version = "0.0.6"
+version = "0.0.7"
```

Comment:
Should all the crates be "0.0.7" now?

Reply:
yep because they're all pinned to the exact version :)